src/components/addcar/hooks/InputChange.jsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+function InputChange(initialstate) {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const InputChange= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/223578657-28c7a2ba-32f4-4a7b-baa1-7d373032f660.png)